iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -659,19 +659,43 @@ popo::InterfacePortData* PortManager::acquireInterfacePortData(capro::Interfaces
     }
 }
 
+void PortManager::publishCurrentServiceRegistry() const noexcept
+{
+    // Send the new serviceRegistry here and ring the bell to inform all ServiceDiscovery instances about the change
+    if (m_serviceRegistryPublisherPortData.has_value())
+    {
+        PublisherPortUserType publisher(m_serviceRegistryPublisherPortData.value());
+        auto maybeChunkHeader = publisher.tryAllocateChunk(sizeof(ServiceRegistry),
+                                                           alignof(ServiceRegistry),
+                                                           CHUNK_NO_USER_HEADER_SIZE,
+                                                           CHUNK_NO_USER_HEADER_ALIGNMENT);
+
+        if (!maybeChunkHeader.has_error())
+        {
+            auto sample = static_cast<ServiceRegistry*>(maybeChunkHeader.value()->userPayload());
+
+            // Copy the complete registry
```

Comment:
Yes, the next line exactly states that ... please remove the comment.

Reminder: A comment should explain the why and not the what. The code already does this.